src/components/DonationTable/DonationTable.tsx

```diff
@@ -109,11 +110,14 @@ export interface DonationRow {
 
 export const createDonationRow = (
   data: DonationTableRowFragment,
+  isOnContactPage: boolean,
```

Comment:
this could be called `hideDisplayName`

Reply:
@canac What are your thoughts? 

Keep it as it is, or change it? 

`isOnContactPage` helps you understand on a high-level that functionality will be different on the contact page.
`hideDisplayName` helps you understand on what the variable will do.

Reply:
Great question! I think I like `hideDisplayName` or `hideAccountDisplayName` better.